src/services/montage-helper.service.js

```diff
@@ -5,7 +5,27 @@
 		.module('montage')
 		.factory('montageHelper', montageHelper);
 
-	function montageHelper(montage, authService) {
+	function montageHelper(montage, authService, ngProgressFactory) {
+
+		var count = 0;
```

Comment:
Use a more descriptive variable name. For example, `pendingRequests` or `pendingRequestCount`.